support-frontend/assets/components/productPage/productPageHero/productPageHero.jsx

```diff
@@ -55,14 +54,27 @@ const HeroHanger = ({
   </div>
 );
 
+const HeroHeading = ({
+  children,
+  hasCampaign,
+}: {children: Node, hasCampaign: boolean}) => (
+  <div className={classNameWithModifiers('component-product-page-hero-heading', [hasCampaign ? 'campaign' : null])}>
+    <LeftMarginSection>
+      {children}
+    </LeftMarginSection>
+  </div>
+);
+
 const ProductPageHero = ({
-  overheading, heading, content, modifierClasses, children, appearance,
+  overheading, heading, content, modifierClasses, children, appearance, hasCampaign,
 }: PropTypes) => (
   <header>
     <HeroWrapper {...{ modifierClasses, appearance }}>
       {children}
-      <HeadingBlock overheading={overheading} >{heading}</HeadingBlock>
     </HeroWrapper>
+    <HeroHeading {...{ hasCampaign }}>
+      <HeadingBlock overheading={overheading} >{heading}</HeadingBlock>
+    </HeroHeading>
```

Comment:
looking at this now it'd make sense to me to have heroHeading make the `HeadingBlock` itself rather than taking it as a children, so you would type
```
<HeroHeading overheading={overheading} heading={heading} hasCampaign={ hasCampaign } />
```

rather than fiddling with a nested HeadingBlock (1/1)

Reply:
Actually - this breaks other stuff which I don't understand. Let's talk about this when I see you next.